client/asset/eth/nodeclient_harness_test.go

```diff
@@ -844,7 +881,15 @@ func testTokenBalance(t *testing.T) {
 	if bal == nil {
 		t.Fatalf("empty balance")
 	}
-	spew.Dump(bal)
+
+	fmt.Println("### Balance:", simnetAddr, stringifyTokenBalance(simnetTokenContractor, bal))
+}
+
+func stringifyTokenBalance(ci tokenContractor, evmBal *big.Int) string {
+	atomicBal := masterToken.EVMToAtomic(evmBal)
+	ui, _ := asset.UnitInfo(testTokenID)
+	prec := math.Round(math.Log10(float64(ui.Conventional.ConversionFactor)))
+	return strconv.FormatFloat(float64(atomicBal)/prec, 'f', int(prec), 64)
```

Comment:
Whoops. Should be `float64(atomicBal)/prec -> float64(atomicBal)/float64(ui.Conventional.ConversionFactor)`

Reply:
As precision?

Reply:
As the first argument to `FormatFloat`. We don't divide by `6`, we divide by `1e6`.

Reply:
Ok, looks right.

Reply:
Corrected.